shared/styles/mixins.scss

```diff
@@ -77,19 +76,42 @@
   }
 }
 
+@mixin v-scale($property, $number) {
```

Comment:
I agree with everything Geir said.

My only question is, is the name `v-scale` confusing or misleading? While the `--scale-element` value is based off the vertical height, there is nothing about this mixin that prevents you from applying it to a horizontal property of something; I suspect that wouldn't be uncommon (to scale things with a know width but without a known height).

In fact, if you were so inclined, you could use this on non-dimensional properties like `opacity` too, though that would be very weird choice.